Mathlib/Algebra/BigOperators/Group/Finset/Defs.lean

```diff
@@ -228,50 +228,101 @@ elab_rules : term
 
 end deprecated
 
-open Lean Meta Parser.Term PrettyPrinter.Delaborator SubExpr
+open Lean Meta Parser.Term PrettyPrinter.Delaborator SubExpr Qq
 open scoped Batteries.ExtendedBinder
 
+/-- For a finset of the form `{x ∈ s | p x}`, returns `(s, some p)`. For any other finset `s`,
+returns `(s, none)`. -/
+def getFinsetFilter {u : Level} {α : Q(Type u)} (s : Q(Finset $α)) :
```

Comment:
It looks like we don't need this, right?

Reply:
Oh indeed, not anymore